/**:
  ros__parameters:
    origin_x: 368305.0700699703
    origin_y: 3278357.100811797
    origin_z: 26.90878
    origin_r: 0.6021965548550632
    utm_zone: 17
    utm_band: 'R'
